build.sh

```diff
@@ -87,11 +104,11 @@ for flavour in win32-ia32 win32-x64 win32-arm64v8; do
   fi
 done
 
-# Linux (x64, ARMv6, ARMv7, ARM64v8)
-for flavour in linux-x64 linuxmusl-x64 linux-armv6 linux-armv7 linux-arm64v8 linuxmusl-arm64v8; do
+# Linux (x64, ARMv6, ARMv7, ARM64v8, s390x)
+for flavour in linux-x64 linuxmusl-x64 linux-armv6 linux-armv7 linux-arm64v8 linuxmusl-arm64v8 linux-s390x; do
   if [ $PLATFORM = "all" ] || [ $PLATFORM = $flavour ]; then
     echo "Building $flavour..."
     docker build -t vips-dev-$flavour $flavour
-    docker run --rm -e "VERSION_VIPS=${VERSION_VIPS}" -e VERSION_LATEST_REQUIRED -v $PWD:/packaging vips-dev-$flavour sh -c "/packaging/build/lin.sh"
+    docker run -it --rm -e "VERSION_VIPS=${VERSION_VIPS}" -e VERSION_LATEST_REQUIRED=false -v $PWD:/packaging:z vips-dev-$flavour sh -c "/packaging/build/lin.sh"
```

Comment:
The `VERSION_LATEST_REQUIRED` env var should be passed through as-is.

Reply:
D'oh! I missed removing this.